frontend/src/index.css

```diff
@@ -29,3 +29,7 @@ html {
 	background: var(--main-background);
 	color: var(--main-text-colour);
 }
+
+*:focus {
+	outline: 0.3rem solid var(--accent-border-colour);
+}
```

Comment:
not seen that before! Pretty snazzy

Reply:
Should this be using `:focus-visible` instead of `:focus`?

Also, 0.3rem is potentially a bit messy if our base font is 16px, whereas 0.3125rem is equivalent to 5px.

Reply:
Ah, just saw your later comment about focus-visible! Ignore that part.